images/installer/Dockerfile.upi.ci.rhel8

```diff
@@ -42,13 +42,13 @@ RUN yum update -y && \
 RUN pip-2 install pyopenssl
 ENV CLOUDSDK_PYTHON=/usr/bin/python
 
-ENV TERRAFORM_VERSION=0.12.24
+ENV TERRAFORM_VERSION=1.0.11
```

Comment:
Can you move this into a separate commit and write a commit message for each.

Reply:
Done. The change to the RHEL Dockerfile was just to gain parity with the main Dockerfile.

Reply:
I meant the terraform version, that is unrelated to the matchbox version, right? 

Can you split the terraform version into a different commit, unless it is related. If it is related please note that in the commit message. 

Can you write a commit message for each like https://github.com/openshift/installer/blob/master/CONTRIBUTING.md#commit-message-format

Reply:
This was originally changed here: https://github.com/openshift/installer/pull/5762/files and the reconciliation didn't happen on the RHEL Dockerfile in that PR. I added it here.

In any case, I'll redo the commit as you pointed out.

Reply:
Got it. We should have caught it in that PR review. Thanks for taking care of it here!